plan.sh

```diff
@@ -38,6 +38,6 @@ do_check() {
 
 do_install() {
   do_default_install
-  gem update --system 2.7.9 --no-document
+  gem update --system 3.2.3 --no-document
```

Comment:
We should really stop install rubygems over the version in Ruby. This makes our hab packages different than a stock ruby install or an omnibus install.